languages.yaml

```diff
@@ -1071,6 +1071,38 @@ Hcl:
       - "#5F43E9"
       - "#4040B2"
     chip: "#AACE60"
+Hlsl:
+  type: programming
+  ascii: |
+    {0}                  $$
+    {0}               $$$$$$$$
+    {0}          $$$$$$$$$$$$$$$$$$         $$$
+    {0} $$$$$$$$$$$$$$$$$$$$$$$$$$$$      $$$$$
+    {0}${1}$$$$${0}$$$$$$$$$$$$$$$$$$$$$$$$$    $${1}$
+    {0}$$     $$$$$$$$$$$$$$$$$$$$$$$$$ $$$${1}$
+    {0}$$     $$$$$$$$$$$$$$$$$$$$$$$$$$$${1}$$$
+    {0}$$$$   $$$$$$$$$$$$$$$$$$$$$$$$$$${1}$$
+    {0} $$$$$$$$$$$$$$$$$$$$$$$$$$$$$$${1}$$$
+    {0}   $$$$$$$$$$$$$$$$$$$$$$$$$${1}$$$$$
+    {0}     $$$$$$$$$$$$$$$$$$$${1}$$$$$$$
+    {0}      {1}$$$$$$$$$$$$${1}$$$$$$$$$$$
+    {0}         {1}$$$$$$$$$$$$$$$$$$
+    {2}      _      _  _        _____  _   
+    {2}     |:|    |:||:|      /:::::||:|
+    {2}     |:|____|:||:|     |:(___  |:|
+    {2}     |::____::||:|      \___:\ |:|
+    {2}     |:|    |:||:|____  ____):||:|____
+    {2}     |_|    |_||______||_____/ |______|
+  colors:
+    ansi:
+      - green
+      - green
+      - white
+    hex:
+      - "#00FF00"
+      - "#006600"
+      - "#FFFFFF"
+    chip: "#AACE60"
```

Comment:
Hi @progDes007, the teapot looks great! 😄 Although I understand your reasoning behind using it as an emblem for this language, given the absence of an official logo for HLSL, it still feels somewhat arbitrary to me. Personally, I'm not particularly fond of the text below the teapot either, especially considering that we already use a similar design in the glsl logo. Considering HLSL is a Microsoft language, I'd like to offer a suggestion:

![image](https://github.com/o2sh/onefetch/assets/13710835/99fb775f-8667-48df-965b-af5223104e7b)

```Hlsl:
  type: programming
  ascii: |
    {0}████████████████  {1}████████████████
    {0}█████  ██  █████  {1}█████  █████████
    {0}█████  ██  █████  {1}█████  █████████
    {0}█████      █████  {1}█████  █████████
    {0}█████  ██  █████  {1}█████  █████████
    {0}█████  ██  █████  {1}█████      █████
    {0}████████████████  {1}████████████████

    {2}████████████████  {3}████████████████
    {2}█████      █████  {3}█████  █████████
    {2}█████  █████████  {3}█████  █████████
    {2}█████      █████  {3}█████  █████████
    {2}█████████  █████  {3}█████  █████████
    {2}█████      █████  {3}█████      █████
    {2}████████████████  {3}████████████████
  colors:
    ansi:
      - red
      - green
      - blue
      - yellow
    hex:
      - "#F65314"
      - "#7CBB00"
      - "#00A1F1"
      - "#FFBB00"
    chip: "#AACE60"

```

Reply:
@o2sh Added the changes as you suggest